distros/debian10/install_phpmyadmin.sh

```diff
@@ -0,0 +1,89 @@
+#---------------------------------------------------------------------
+# Function: InstallphpMyAdmin
+#    Install and configure phpMyAdmin
+#---------------------------------------------------------------------
+InstallphpmyAdmin() {
+    echo -n "Install myPhpAdmin...."
+    mkdir /usr/share/phpmyadmin
+    mkdir /etc/phpmyadmin
+    mkdir -p /var/lib/phpmyadmin/tmp
+    chown -R www-data:www-data /var/lib/phpmyadmin
+    touch /etc/phpmyadmin/htpasswd.setup
+    cd /tmp
+    wget https://files.phpmyadmin.net/phpMyAdmin/4.9.0.1/phpMyAdmin-4.9.0.1-all-languages.tar.gz
+    tar xfz phpMyAdmin-4.9.0.1-all-languages.tar.gz
+    mv phpMyAdmin-4.9.0.1-all-languages/* /usr/share/phpmyadmin/
+    rm phpMyAdmin-4.9.0.1-all-languages.tar.gz
+    rm -rf phpMyAdmin-4.9.0.1-all-languages
+    cp /usr/share/phpmyadmin/config.sample.inc.php  /usr/share/phpmyadmin/config.inc.php
+    sed -i "$cfg['blowfish_secret'] = 'bD3e6wva9fnd93jVsb7SDgeiBCd452Dh'; /* YOU MUST FILL IN THIS FOR COOKIE AUTH! */" /usr/share/phpmyadmin/config.inc.php
+    sed -i "$cfg['TempDir'] = '/var/lib/phpmyadmin/tmp';" /usr/share/phpmyadmin/config.inc.php
+    echo -n " create apache configuration file for myPHPadmin"
```

Comment:
Better would be:

"Creating Apache config file for PhpMyAdmin"

Reply:
done.